Assets/Editor/BuildTiltBrush.cs

```diff
@@ -110,9 +110,12 @@ public BuildFailedException(string message)
         {
             // Mono
             new KeyValuePair<XrSdkMode, BuildTarget>(XrSdkMode.Monoscopic, BuildTarget.StandaloneWindows64),
+            new KeyValuePair<XrSdkMode, BuildTarget>(XrSdkMode.Monoscopic, BuildTarget.StandaloneOSX),
+            new KeyValuePair<XrSdkMode, BuildTarget>(XrSdkMode.Monoscopic, BuildTarget.Android),
 
             // OpenXR
             new KeyValuePair<XrSdkMode, BuildTarget>(XrSdkMode.OpenXR, BuildTarget.StandaloneWindows64),
+            new KeyValuePair<XrSdkMode, BuildTarget>(XrSdkMode.OpenXR, BuildTarget.StandaloneOSX),
```

Comment:
If this is a valid combination, can you add it to the CI please?

Reply:
I will do once it's working. Still very much WIP

Reply:
Ah. I see what I've done. I made the PR and then continued working on stuff, forgetting about the PR. I should create a new branch to separate out PR-ready from WIP. My mistake.